tests/rest/datacube/test_datacube.py

```diff
@@ -1106,25 +1110,6 @@ def test_save_result_and_create_job_at_most_one_with_format(
             "result": True,
         }
 
-    @pytest.mark.parametrize(
-        ["save_result_format", "execute_format"],
-        [
-            ("NetCDF", "NetCDF"),
-            ("GTiff", "NetCDF"),
-        ],
-    )
-    def test_save_result_and_create_job_both_with_format(self, s2cube, save_result_format, execute_format):
-        cube = s2cube.save_result(format=save_result_format)
-        with pytest.raises(
-            OpenEoClientException,
-            match=re.escape(
-                "DataCube.create_job() with explicit output format 'NetCDF',"
-                " but the process graph already has `save_result` node(s)"
-                " which is ambiguous and should not be combined."
-            ),
-        ):
-            cube.create_job(out_format=execute_format)
```

Comment:
This test case should be kept (this use case should give a better error than the generic one)